eviloauth/dispatcher.py

```diff
@@ -1,11 +1,80 @@
 import sys
 import logging
+from enum import Enum, auto
+from diskcache import Cache
 from eviloauth.idp import IDP
 from eviloauth.exceptions import EviloauthCommandException
+from eviloauth.access_token import AccessToken
+from eviloauth import app, cache, temp_dir, load_modules
+from datetime import datetime, timedelta
+from eviloauth.general_token import GeneralToken
+
+
+class CommandType(Enum):
+    EXIT = "exit"
+    MODULE = "module"
+    TOKENS = "tokens"
+    IDP = "idp"
+    TARGET = "target"
+    URL = "url"
+    HELP = "help"
+
+
+class TokenSubCommand(Enum):
+    LIST = "list"
+    DELETE = "delete"
+
+
+class IDPSubCommand(Enum):
+    LIST = "list"
+    CONFIGURE = "configure"
+
+
+class PlatformType(Enum):
+    AZURE = "azure"
+    AWS = "aws"
+    OKTA = "okta"
+
+
+class AzureAttack(Enum):
+    READ_MAIL = "read_mail"
+
+
+class TargetSubCommand(Enum):
+    LIST = "list"
+    SET = "set"
+
+
+class TokenManager:
```

Comment:
This needs to be in a different file. One class per file, move it to token_manager.py